### PR TITLE
LPD-27459 Test accessibility in Playwright

### PR DESCRIPTION
**Pull request by veroglez**

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by veroglez** [comment]
ci:test:sf

File: modules/test/playwright/utils/checkAccessibility.ts [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+	bestPractices = false,
+	page,
+	selectors,
+	selectorsToExclude,
+}: Props) {
+	const tags = bestPractices ? [...TAGS, 'best-practice'] : TAGS;
+
+	if (selectors) {
+		for (const locator of selectors) {
+			page.locator(locator).waitFor();
```
Comment (ethib137) [inline modules/test/playwright/utils/checkAccessibility.ts:40]: @veroglez what's this needed for?

Comment (veroglez) [reply modules/test/playwright/utils/checkAccessibility.ts]: It's to make sure that the locators we want to analyze are on the page at the time the `.analyze()` is executed. It is something that the playwright recommends [https://playwright.dev/docs/accessibility-testing](https://playwright.dev/docs/accessibility-testing#:~:text=//%20It%20is%20important%20to%20waitFor()%20the%20page%20to%20be%20in%20the%20desired%0A%20%20//%20state%20*before*%20running%20analyze().%20Otherwise%2C%20axe%20might%20not%0A%20%20//%20find%20all%20the%20elements%20your%20test%20expects%20it%20to%20scan)

**Comment by sandrodw3** [comment]
This is awesome @veroglez 😍 with this we'll be able to save a lot of code analyzing accessibility in comparison to what we did in poshi, thanks for the idea!

The PR LGTM, from my side it's ready to go 🚀

**Comment by ealonso** [comment]
ci:forward

**Pull request closed without merging**